shared/js/views.jsx

```diff
@@ -631,13 +663,17 @@ loop.shared.views = (function(_, mozL10n) {
         return;
       }
 
-      var videoElement = this.getDOMNode();
+      var videoElement = this.getDOMNode().querySelector("video");
 
-      if (videoElement.tagName.toLowerCase() !== "video") {
+      if (!videoElement || videoElement.tagName.toLowerCase() !== "video") {
         // Must be displaying the avatar view, so don't try and attach video.
         return;
       }
 
+      if (this.props.shouldRenderRemoteCursor) {
+        videoElement.addEventListener("loadeddata", this.handleVideoUpdate);
```

Comment:
I believe this will only trigger once, so it will only record the initial `<video>` element size and the initial stream size. This results in wrong cursor positions if either the link clicker or link generator resizes the window. For the latter, I'm not sure if there's a handy "streamresized" event, so if not, we could send height and width as part of the cursor data.